jumpscale/packages/tfgrid_solutions/chats/ubuntu_deploy.py

```diff
@@ -187,14 +187,9 @@ def container_pay(self):
                 channel_port=self.user_form_data["Logs Channel port"],
                 channel_name=self.user_form_data["Logs Channel name"],
             )
-        metadata = dict()
-        metadata["chatflow"] = self.user_form_data["chatflow"]
-        metadata["Solution name"] = self.user_form_data["Solution name"]
-        metadata["Version"] = self.user_form_data["Version"]
-        metadata["Solution expiration"] = self.user_form_data["Solution expiration"]
```

Comment:
why this was removed ? duplicate ?

Reply:
no need to filter self.user_form_data , just send it yes